Frameworks/UIKit/UIActivityIndicatorView.mm

```diff
@@ -226,28 +228,32 @@ - (UIActivityIndicatorViewStyle)activityIndicatorViewStyle {
  @Status Interoperable
 */
 - (void)startAnimating {
-    if (_isAnimating) {
-        return;
-    }
+    RunSynchronouslyOnMainThread(^{
```

Comment:
Many apps don't bother to check, and appear to work fine on the reference platform.  We already do this for UIView init/dealloc and in a few other spots (for sone middleware scenarios), but as called out in the commit message, we may revisit this in a future task that addresses the issue as a whole.